DEVELOPMENT.md

```diff
@@ -107,10 +113,13 @@ diffs.
 
 ### Running the adapter on your local machine
 
-Sometimes you might want to develop against a VSphere server that is not
-accessible from your development cluster. So you can run the receive adapter
-(the data plane for the events) locally like so. Note that you will still need a
-kubernetes cluster to use for the ConfigMap-based bookmarking
+Sometimes you might want to develop against a VMware vSphere environment that is
+not accessible from your development cluster. You can run the receive adapter
+(the data plane for the events) locally in such cases (as opposed to running it
+within a Kubernetes environment). 
+
```

Comment:
Remove the trailing space for the ci to rerun and hopefully (if the flakes also goes away) we can merge afterwards